tests/kubeless-test-client/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM debian:stable-slim
+FROM alpine:latest
 
 LABEL source="git@github.com:kyma-project/kyma.git"
 
-RUN apt-get update && apt-get install -y curl unzip && rm -r /var/lib/apt/lists/*
+RUN apk add --no-cache curl jq
 
-RUN curl -Lo /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/linux/amd64/kubectl && chmod +x /usr/local/bin/kubectl
+RUN curl -Lo /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x /usr/bin/kubectl
 
-RUN curl -Lo /tmp/kubeless.zip https://github.com/kubeless/kubeless/releases/download/v1.0.0-alpha.2/kubeless_linux-amd64.zip && unzip -uq /tmp/kubeless.zip -d /tmp/ && mv /tmp/bundles/kubeless_linux-amd64/kubeless /usr/local/bin/ && rm -r /tmp/kubeless.zip /tmp/bundles && chmod +x /usr/local/bin/kubeless
+RUN curl -Lo /tmp/kubeless.zip "$(curl -s https://api.github.com/repos/kubeless/kubeless/releases/latest | jq -r '.assets[]|select(.name=="kubeless_linux-amd64.zip").browser_download_url')" && unzip -q /tmp/kubeless.zip -d /tmp/ && mv /tmp/bundles/kubeless_linux-amd64/kubeless /usr/bin/ && rm -r /tmp/kubeless.zip /tmp/bundles && chmod +x /usr/bin/kubeless
```

Comment:
IMO versions should be explicit in order to keep the tests deterministic

Reply:
Changed.